### PR TITLE
NEWS: Update sockets provider notes for v1.1.1 and v1.1.0

### PR DESCRIPTION
**Pull request by shantonu**
Signed-off-by: Shantonu Hossain <shantonu.hossain@intel.com>

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -14,6 +14,20 @@ v1.1.1, TBD
 
 ## Sockets provider notes
 
+- General clean-up and restructuring
+- Added fallback mechanism for getting soirce address
```
Comment (shefty) [inline NEWS.md]: spelling - source

File: NEWS.md [hunk 2]
```diff
@@ -14,6 +14,20 @@ v1.1.1, TBD
...
+- Fixed fi_getinfo to use user provided capabilities from hints
+- Fixed hostname and port number and added checks in sock_av_insertsym
+- Add retry for connection timeout
+- Release av resources in the error path
+- Removed separate read/write CQ to be consistent with the man page
+- Increased default connection map size and added environment variable to specify
+AV, CQ, EQ and connection map size to run large scale tests
```
Comment (shefty) [inline NEWS.md]: indentation

File: NEWS.md (hunk 1 above)
Comment (jsquyres) [inline NEWS.md]: No, it's ok -- it's the northeast spelling.

:wink:

**Comment by bturrubiates** [comment]
This should also be getting merged into master, right?

**Comment by shantonu** [comment]
Yes, will do.

**Comment by jithinjosepkl** [comment]
@shantonu - can you update the NEWS to add info about #1317 ?

**Comment by shantonu** [comment]
@jithinjosepkl, done!
@shefty, fixed the PR and added notes for v1.1.0

**Comment by shefty** [comment]
Looks good

**Pull request merged**